sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -514,6 +612,21 @@ public void run() {
 	                		Log.e(TAG, "No reply address included, can't send a reply");
 	                	}
 	                	break;
+	                case TransportConstants.ROUTER_REQUEST_SECONDARY_TRANSPORT_CONNECTION:
+	                	// Currently this only handles one TCP connection
```

Comment:
This assumes that the secondary transport is always TCP, but Core could send the transport priority in any given order.

Reply:
This is actually correct logic. This message is specifically for requesting the router service to attempt to connect to a transport. It is impossible to connect USB simply by software, and the bluetooth transport will already be listening when applicable but it does not connect out to core.